addons/partner_autocomplete/models/res_partner_autocomplete_sync.py

```diff
@@ -14,25 +14,8 @@ class ResPartnerAutocompleteSync(models.Model):
     synched = fields.Boolean('Is synched', default=False)
 
     @api.model
-    def start_sync(self):
-        to_sync_items = self.search([('synched', '=', False)])
-        for to_sync_item in to_sync_items:
-            partner = to_sync_item.partner_id
-
-            params = {
-                'partner_gid': partner.partner_gid,
-            }
-
-            if partner.vat and partner._is_vat_syncable(partner.vat):
-                params['vat'] = partner.vat
-                _, error = self.env['iap.autocomplete.api']._request_partner_autocomplete('update', params)
-                if error:
-                    _logger.warning('Send Partner to sync failed: %s', str(error))
-
-            to_sync_item.write({'synched': True})
+    def start_sync(self, batch_size=1000):
```

Comment:
The `batch_size` parameter wasn't present in 17.0.
```suggestion
    def start_sync(self):
```